frontend/resourceadm/components/AccessListMembers/AccessListMembersTable.tsx

```diff
@@ -42,7 +43,7 @@ export const AccessListMembersTable = ({
         {listItems.map((item) => {
           return (
             <Table.Row key={item.orgNr}>
-              <Table.Cell>{item.orgNr}</Table.Cell>
+              <Table.Cell aria-label={item.orgNr.split('').join(' ')}>{item.orgNr}</Table.Cell>
```

Comment:
I would recommend to create the aria-label value as a variable to make it more readable and easier to refactor later 😄